From 6f5668f2b419d1af201fc844dd1c0507f00c97f5 Mon Sep 17 00:00:00 2001
From: Philip Yang <Philip.Yang@amd.com>
Date: Mon, 23 Oct 2017 13:09:54 -0400
Subject: [PATCH 2132/4131] drm/amdkfd: remove unused label to get rid of
 warning message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

 warning: label ‘evict_fence_fail’ defined but not used [-Wunused-label]
 evict_fence_fail:

Change-Id: I755428b74962312b7fe21fec3f9aee1345464bce
Signed-off-by: Philip Yang <Philip.Yang@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index 5de8398..39b71cd 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -2404,12 +2404,12 @@ int amdgpu_amdkfd_gpuvm_restore_process_bos(void *info)
 
 		amdgpu_bo_fence(bo, &process_info->eviction_fence->base, true);
 	}
+
 validate_map_fail:
 	ttm_eu_backoff_reservation(&ctx.ticket, &ctx.list);
 	amdgpu_sync_free(&sync_obj);
 ttm_reserve_fail:
 	mutex_unlock(&process_info->lock);
-evict_fence_fail:
 	kfree(pd_bo_list);
 	return ret;
 }
-- 
2.7.4

